<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Abhi1996" failures="2" tests="2" name="Test(failed)" time="27.665" errors="0" timestamp="2025-06-02T12:29:59 IST">
  <testcase classname="testCases.TC_03_LoginDDT" name="VerifyLoginDDT" time="12.219">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC_03_LoginDDT.VerifyLoginDDT(TC_03_LoginDDT.java:47)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- VerifyLoginDDT -->
  <testcase classname="testCases.TC_03_LoginDDT" name="VerifyLoginDDT" time="2.988">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC_03_LoginDDT.VerifyLoginDDT(TC_03_LoginDDT.java:52)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- VerifyLoginDDT -->
</testsuite> <!-- Test(failed) -->
